internal/nginx/nginx.go

```diff
@@ -23,6 +23,7 @@ type Controller struct {
 	nginxConfdPath        string
 	nginxSecretsPath      string
 	local                 bool
+	debug                 bool
```

Comment:
better to have a field initialized either with `nginx` or `nginx-debug`. You can use that field in the `Start` method as well as in `Reload` and `Quit` (for consistency). 

You can change `NewNginxController` to take a string parameter for the binary Path, similarly to the `nginxConfPath` instead of having `debug bool` parameter.